tests/backend-storage/src/common-tests/Config.ts

```diff
@@ -10,6 +10,10 @@ interface StorageIntegrationTestsConfig {
   clientStorage: ClientStorage;
 }
 
+interface InternalStorageIntegrationTestsConfig {
+  serverStorage: ServerStorage;
```

Comment:
Symbols in this file are unused because of the refactoring

Reply:
Removed